### PR TITLE
Add exercise on process management "security breach"

### PR DESCRIPTION
**Pull request by eskultety**

### CODE REVIEW

File: exercises/lesson_05/breach/README.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    # Watch the wonderful intro
+    $ ./lab intro
+
+    # Resolve the tasks
+
+    # Grade your work
+    $ sudo ./lab grade
+
+    # Clean the environment that was created for the lab    (requires root)
+    $ sudo ./lab cleanup
```
Comment (Aritma) [inline exercises/lesson_05/breach/README.rst:26]: Would be nice to add info, how to end the lab (if possible) if you either cannot or don't want to finish the lab.

File: exercises/lesson_05/breach/lab [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+
+cleanup()
+{
+    cd data/ansible
+    ansible-playbook cleanup.yml
+
+    while read line; do
+        yum remove -y $line
+    done < $TEMPFILE
```
Comment (Aritma) [inline exercises/lesson_05/breach/lab:80]: If I understand correctly, student is able to delete this file. Is it save to store data so transparently in /tmp? If student deletes TEMPFILE, script won't clear the system properly.

File: exercises/lesson_05/breach/README.rst (hunk 1 above)
Comment (eskultety) [reply exercises/lesson_05/breach/README.rst:26]: Hmm, you'd still run cleanup - isn't it self-explanatory?

File: exercises/lesson_05/breach/lab (hunk 2 above)
Comment (eskultety) [reply exercises/lesson_05/breach/lab:80]: True, I'll put it in /var

**Pull request merged**